Makefile

```diff
@@ -120,6 +114,10 @@ DOCKER_ARCH_TAG=$(ZARCH)
 
 FULL_VERSION:=$(ROOTFS_VERSION)-$(HV)-$(ZARCH)
 
+# must be included after ZARCH is set
+include $(CURDIR)/kernel-version.mk
+$(info "Building $(ZARCH) image with kernel $(KERNEL_VERSION) from branch $(KERNEL_BRANCH) and commit $(KERNEL_COMMIT)")
```

Comment:
I'm not sure if it makes sense to put this info message here, because if the user defines its own KERNEL_TAG, so the KERNEL_{VERSION,BRANCH,COMMIT} might not correspond to the custom TAG, right? In that case, it can create some confusion and misinformation...

Reply:
ah, yes. I forgot to remove it